### ISSUE TITLE
Staging Review finding: Claims process graphic cut off at high zoom

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#88737]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits team 1
**Product:** Claim status tool
**Feature:** Claims process overview page

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On the Overview page ([example](https://staging.va.gov/track-claims/your-claims/600519576/overview)), the graphic depicting the 8 steps of the claim process is cut off at high zoom and/or on some mobile devices. At 400% zoom (equivalent to a viewport of 320px), step 8 is off the page.

The actual impact of this is minimal. But WCAG guidance requires content be visible up to 400% zoom, and it's possible that it could be confusing to a Veteran to read text that says "there are 8 steps" and then see a graphic that appears to show only 7 steps. 

<details><summary>Link, screenshot or steps to recreate</summary>

![Seven steps in the claims process graphic, with the eighth hidden beneath the browser scrollbar.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/Brian-71293-1.png?raw=true)

</details>

## Recommended action

I think this has an easy fix!

The graphic is an SVG with `width="352"`, so the last 32px of the image will be cut off on a 320px viewport. But adding a CSS style of `max-width: 100%` appears to resolve the issue without any impact on other viewports.

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** [WCAG 1.4.10 Reflow - Level AA](https://www.w3.org/TR/WCAG21/#reflow)
- **Modality:** Browser zoom
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
